### PR TITLE
Update facet-plots.md

### PR DESCRIPTION
**Pull request by SylwiaOliwia2**
Added section 'Zooming subplots to the same range'. A part of #1965

### CODE REVIEW

File: doc/python/facet-plots.md [hunk 1]
```diff
@@ -118,3 +118,20 @@ fig.show()
 ```python
 
 ```
+
+### Zooming subplots to the same range
+
+Using `facet_col` from `plotly.express` let zoom each facet to the same range impliciltly. However, if the subplots are created with `make_subplots`, the axis needs to be updated with `matches` parameter to zoom all the subplots accordingly. Zoom in one trace below, to see the other subplots zoomed to the same x-axis range:
```
Comment (emmanuelle) [inline doc/python/facet-plots.md]:
```suggestion
Using `facet_col` from `plotly.express` let zoom each facet to the same range implicitly. However, if the subplots are created with `make_subplots`, the axis needs to be updated with `matches` parameter to zoom all the subplots accordingly. Zoom in one trace below, to see the other subplots zoomed to the same x-axis range:
```

File: doc/python/facet-plots.md [hunk 2]
```diff
@@ -118,3 +118,20 @@ fig.show()
 ```python
 
 ```
+
+### Zooming subplots to the same range
```
Comment (emmanuelle) [inline doc/python/facet-plots.md]: `matches` does actually more than synchronizing zoom, you also have synchronized panning for example. How about a more general title like "Synchronizing axes in subplots with `matches`"? Also what do you think of mentioning pan and zoom in the text below (compared to just zoom at the moment?)

**Comment by emmanuelle** [comment]
Thanks a lot @SylwiaOliwia2 for this nice example! The example is great, I left a suggestion on how to make the text a little more general, please tell me what you think.

File: doc/python/facet-plots.md (hunk 2 above)
Comment (SylwiaOliwia2) [reply doc/python/facet-plots.md]: Sure! I will edit the title and mention the pan.

**Comment by emmanuelle** [comment]
Thank you very much! Merging :-).

**Pull request merged**